src/model/dcopf.jl

```diff
@@ -298,7 +298,16 @@ function get_egen_gen(data, model, gen_idx, year_idx, hour_idx)
 end
 
 export get_egen_gen
-  
+
+"""
```

Comment:
Instead of doing this, I've found it's faster to pull out the `pcap_gen` variable up front in your `modify_model!` function with a type assert.  i.e.
```julia
pcap_gen = model[:pcap_gen]::Matrix{VariableRef}
```
Then the compiler can infer the type when you access via
```julia
pcap_gen[gen_idx, year_idx]
```